src/librbd/internal.cc

```diff
@@ -1017,8 +1031,8 @@ int mirror_image_disable_internal(ImageCtx *ictx, bool force,
   }
 
   int create_v2(IoCtx& io_ctx, const char *imgname, uint64_t bid, uint64_t size,
-		int order, uint64_t features, uint64_t stripe_unit,
-		uint64_t stripe_count, uint8_t journal_order,
+                int order, uint64_t features, uint64_t stripe_unit,
```

Comment:
No need to keep the sync version -- the only person going to call it is the rbd CLI (via the API) so that's safe.  It's potentially not safe if an async state machine invokes a blocking sync state machine.